docs/how_to/create_custom_coords.rst

```diff
@@ -0,0 +1,38 @@
+.. _sunpy-how-to-create-custom-coordinate-objects:
+
+**********************************************
+Create Coordinate Objects with Custom Observer
+**********************************************
+
+It is possible to specify an observer other than Earth by providing a `~astropy.coordinates.SkyCoord` object representing the observer's location.
+This can be useful when working with data from instruments located at specific points in space, such as spacecraft.
+
+.. code-block:: python
+
+    >>> from astropy.coordinates import SkyCoord, CartesianRepresentation
+    >>> from sunpy.coordinates import frames
+    >>> import astropy.units as u
+
+Define the observer's position in Heliocentric coordinates
+
+.. code-block:: python
+
+    >>> observer_coord = SkyCoord(-100*u.arcsec, 500*u.arcsec, frame=frames.Helioprojective)
+    >>> observer_coord= SkyCoord(x=-72241.0*u.km, y=361206.1*u.km, z=589951.4*u.km, frame=frames.Heliocentric)
+    >>> observer_coord = SkyCoord(70*u.deg, -30*u.deg, frame=frames.HeliographicStonyhurst)
+    >>> observer_coord
+    <SkyCoord (HeliographicStonyhurst: obstime=None, rsun=695700.0 km): (lon, lat) in deg
+        (70., -30.)>
```

Comment:
```suggestion
    >>> observer_coord = SkyCoord(70*u.deg, -30*u.deg, 1*u.AU, frame=frames.HeliographicStonyhurst)
    >>> observer_coord
    <SkyCoord (HeliographicStonyhurst: obstime=None, rsun=695700.0 km): (lon, lat, radius) in (deg, deg, AU)
        (70., -30., 1.)>
```
If you don't specify a distance to `HeliographicStonyhurst`, it will get interpreted as being on the surface of the Sun, which again is not a realistic observer location.

Reply:
Ah, Understood